governance/microservice.go

```diff
@@ -300,6 +301,11 @@ func (w *GovernanceWorker) GetEnvVarsForServiceDepolyment(msdef *persistence.Mic
 	envAdds[config.ENVVAR_PREFIX+"ORGANIZATION"] = exchange.GetOrg(w.GetExchangeId())
 	envAdds[config.ENVVAR_PREFIX+"PATTERN"] = w.devicePattern
 	envAdds[config.ENVVAR_PREFIX+"EXCHANGE_URL"] = w.Config.Edge.ExchangeURL
+	envAdds[config.ENVVAR_PREFIX+"ESS_API_PROTOCOL"] = w.BaseWorker.Manager.Config.GetFileSyncServiceProtocol()
```

Comment:
Get rid of line 301-303 and just call SetPlatformEnvvars(...). I think you can update PlatformEnvvars to remove the HASH env var. That will make all the code consistent. hzn dev service start also uses PlatformEnvvars to set the ESS variables.